intro_to_dygraph.Rmd

```diff
@@ -0,0 +1,116 @@
+# An Introduction to Dygraph in R
+
+Yanyun Chen
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(warning = FALSE, message = FALSE)
+```
+
+
+### These are the codes demonstrated in the video tutorial. The video tutorial can be found at https://www.youtube.com/watch?v=ewQdzu47yZs
+
+
+***Data Source: FRED, https://fred.stlouisfed.org/categories/9?t=cpi&ob=pv&od=desc#***
+
+
+```{r library}
+library(readxl)
+library(dygraphs)
+library(dplyr)
+```
+
+
+### 0. Project Description
```

Comment:
Hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings